### PR TITLE
Minor changes and improvements

### PR DESCRIPTION
**Pull request by Buddybenj**

### CODE REVIEW

File: include/summary-4.tex [hunk 1]
```diff
@@ -18,9 +18,9 @@
 to define an element of~$X$ which, by the very way we define it, is
 guaranteed to be different from every element in the enumeration. So
 the enumeration can't be an enumeration of all of~$X$ after all, and
-we've shown that no enumeration of~$X$ can exist.  A reduction shows
+we've shown that no enumeration of~$X$ can exist. A reduction shows
 that $X$ is uncountable by associating every element of some known
-uncountable set~$Y$ with an element of~$X$ in a bijective way. If this
+uncountable set~$Y$ with an element of~$X$ in an injective way. If this
```
Comment (Buddybenj) [inline include/summary-4.tex]: Please confirm if this change is correct.

Comment (rzach) [reply include/summary-4.tex]: It is correct, but not in line with the section being summarized (although neither was the old version). It should be "by associating every element of~$X$ with an element of some known uncountable set~$Y$ in a surjective way."

File: include/summary-2.tex [hunk 2]
```diff
@@ -1,11 +1,11 @@
 A \emph{relation}~$R$ on a set~$X$ is a way of relating elements
 of~$X$. We write $Rxy$ if the relation \emph{holds} between $x$
 and~$y$. Formally, we can consider $R$ as the sets of pairs
-$\tuple{x,y} \in X^2$ such that~$Rxy$.  Being less than, greater than,
-equal to, evenly dividing, being the same length, being a subset of,
```
Comment (rzach) [inline include/summary-2.tex:5]: Can you leave in the first "being" (the "evenly dividing" sortof cancels the repetition of "being" if that makes sense).

**Review by rzach — CHANGES_REQUESTED** [review]

**Review by rzach — APPROVED** [review]

**Comment by rzach** [comment]
Thanks for suggesting these changes. I'm not sure how GitHub wants me/you to make changes to the pull request. Perhaps by "approving the requested changes" I indicated that the pull request is now ready to merge instead of indicating that I agreed (with myself!) that these changes should be made. Let me know if you see a way to change the pull request (ie change your changes) or if I should do it.

**Comment by Buddybenj** [comment]
I made the requested changes. As long as there are no glaring errors, feel free to merge.

**Comment by rzach** [comment]
Thanks again!

**Pull request merged**